tests/roles/run_tests/tasks/healthcheck.yml

```diff
@@ -102,6 +103,7 @@
     kind: Node
     label_selectors:
       - "node-role.kubernetes.io/control-plane"
+      - "node-role.kubernetes.io/master"
```

Comment:
this is failing since we are filtering out with label selector and looking for both control-plane and master labels to be in the controlplane node